src/clients/mailing-client.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const Client = require('./base-server-client')
+
+class MailingClient {
```

Comment:
shouldn’t we distinguish in name that `MailingClient` is using `base-server-client`? However `MailingServiceClient` does not sounds any good xD